src/index.html

```diff
@@ -1,12 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__start-day-fri calendar__month-length-28">
+      <div class="cell"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--fri calendar--month-length--28">
      <div class="calendar__cell"></div>
```
Make sure that you follow BEM requirements and naming conventions
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/36073dd7-4a18-4ea3-9dbc-c7d27ee6b58c)